backend/app/Chat/Servers/ServerQueue.php

```
@@ -122,6 +122,102 @@ public static function hasAtLeastOnePendingItem(string $user): bool
+
+            $stmt->bindParam(':user_uuid', $user_uuid);
+            $stmt->execute();
+            $result = $stmt->fetch(\PDO::FETCH_ASSOC);
+
+            return (int)($result['count'] ?? 0);
+        } catch (\Exception $e) {
+            self::db_Error('Failed to count pending servers: ' . $e->getMessage());
+
+            return 0;
+        }
+    }
```

PHPMD (2.15.0)

The parameter $user_uuid is not named in camelCase. (undefined)

(CamelCaseParameterName)